doc/ebsnvme-id.8

```diff
@@ -5,32 +5,35 @@
  \" of this license, visit
  \" http://creativecommons.org/licenses/by-sa/4.0/.
  \" SPDX-License-Identifier: CC-BY-SA-4.0
-.TH EBSNVME-ID 8 "May  4 2020"
+.TH EBSNVME-ID 8 "Jan 19 2024"
 .SH NAME
 ebsnvme-id \- Reads and prints EBS information from NVMe devices
 .SH SYNOPSIS
 .B ebsnvme-id
-.RI [ options ] " DEVICE"
+.RI [ options ]
 .br
 .SH DESCRIPTION
 .B ebsnvme-id
-uses the NVME Management Interface to read various metadata about
-NVME-attached EBS storage volumes.  It can be used to retrive the EBS
+uses the NVMe Management Interface to read various metadata about
+NVMe-attached EBS storage volumes.  It can be used to retrive the EBS
 Volume ID or EC2 API device mapping value given the local block device
 name.
 
 .SH OPTIONS
 .TP
-.B \-v, \-\-volume
+.B \-v <device>
```

Comment:
any reason for dropping the long versions of the options?

Reply:
Nothing uses it except humans who don't need to run this anyways and I don't want to parse the command line without regular shell utilities that don't exist in early boot :D